usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -117,6 +117,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/recipient/children/<DUNS\>/](/api/v2/recipient/children/006928857/)|GET| Returns recipient details based on DUNS number |
 |[/api/v2/recipient/duns/<HASH_VALUE\>/](/api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/)|GET| Returns a high-level overview of a specific recipient, given its id |
 |[/api/v2/recipient/duns/](/api/v2/recipient/duns/)|POST| Returns a list of recipients in USAspending DB |
+|[/api/v2/recipient/duns/count/](/api/v2/recipient/duns/count/)|POST| Returns the count of recipents for the given filters |
```

Comment:
I think the path for this is off. The API docs have it as `/api/v2/recipient/count/` ... you have `duns` in there.

Reply:
I have adjusted all these urls to match this format: `/api/v2/recipient/count/`